README.md

```diff
@@ -19,3 +19,4 @@ Installation and administrator documentation for FileSender 2.0 is provided at [
 
 ## Developer
 Development documentation is provided in the [contribution guide](CONTRIBUTE.md).
+
```

Comment:
Ask Jan about access to TravisCI. There's a button there to re-run the build, but i'm not sure you have access to it.